src/lib/npfg/AirspeedReferenceController.cpp

```diff
@@ -0,0 +1,63 @@
+/****************************************************************************
+ *
+ * Copyright (c) 2024 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2025 PX4 Development Team. All rights reserved.
```
saw it in a few other places as well

Reply:
done